src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeProviderImpl.java

```diff
@@ -537,99 +295,85 @@ private JSONArray downloadVehicleDetails(String external_route_id) throws IOExce
         return response;
     }
 
-    private int calcDelayTime(long arrivalTime) {
-        int diff;
-        String pattern = "yyyy-MM-dd'T'HH:mm:ssZ";
-        DateTimeFormatter dtf = DateTimeFormat.forPattern(pattern);
-        DateTime parsedDate = dtf.parseDateTime(responseTimeStamp);
-        diff = (int) (arrivalTime - parsedDate.getMillis() / 1000);
-        return diff;
-    }
-
-    private String convert2FormattedTime(String myTimeStamp) {
-
-        DateTimeFormatter dtf = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ssZ");
-        DateTime jodatime = dtf.parseDateTime(myTimeStamp);
-        DateTimeFormatter dtfOut = DateTimeFormat.forPattern("HH:mm:ss");
-        return dtfOut.print(jodatime);
-    }
-
-    // This method extract time from timestamp
-    private long convertTime(String myTimeStamp) {
-
-        //final SimpleDateFormat sdf = new SimpleDateFormat( "yyyy-MM-dd'T'HH:mm:ssXXX");
-        DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssZ") {
-            public Date parse(String source, ParsePosition pos) {
-                return super.parse(source.replaceFirst(":(?=[0-9]{2}$)", ""), pos);
-            }
-        };
-        Date time;// = new Date();
-        long result = 0;
+    /* Method to get Vehicle locations for MSC Express (has the same route_id with C)

```

Comment:
Could you turn this into a normally-formatted Javadoc block?

See previous comment for how to auto-generate the template.